notebooks/demo.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>Thanks Shrey. This looks nice. Is it possible to increase the character limit per column, so that we can read and compare some answers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes, also adding a csv of results.